### ISSUE TITLE
Add winget (Windows package manager) support for installing lorawan-stack-cli

### ISSUE THREAD

**Issue by lnlp** [TheThingsNetwork/lorawan-stack#4306]
#### Summary
Please add support for Microsoft's official Windows CLI package manager **winget** for installing `lorawan-stack-cli` (`ttn-lw-cli`).

#### Why do we need this?

So Windows 10 users can simply install the TTN CLI tools with: `winget install ttn-lw-cli` 

Similar to how macOS and Linux users can currently install it via their package managers.

The macOS and Linux package managers are already supported so importance of package managers does not need further explanation (neither does the Windows 10 operating system with its 1.3+ billion users).

Until recent there was no official package manager for Windows but now Microsoft has released the Windows CLI package manager **winget**.

`winget` is currently at version v1.0.11451.
It needs to be manually installed but will be added standard to Windows in a near future Windows update. 
For more information see:
- https://github.com/microsoft/winget-cli
- https://github.com/microsoft/winget-cli/blob/master/doc/windows-package-manager-v1-roadmap.md

#### What is already there? What do you see now?

Unlike for macOS and Linux the `winget` package manager for Windows is currently not yet supported.

#### What is missing? What do you want to see?

Support for the `winget` package manager to install the `lorawan-stack-cli` (`ttn-lw-cli`).

#### Environment

Windows 10.

#### How do you propose to implement this?

Information about the Windows Package Manager Community Repository and how to create a package can be found here:
- https://github.com/microsoft/winget-pkgs

#### How do you propose to test this?

Needs no further explanation.

#### Can you do this yourself and submit a Pull Request?

No.

**Comment by johanstokking** [comment]
Thanks @lnlp !

**Comment by neoaggelos** [comment]
Currently, winget requires an installer, which we do not have for Windows builds. There is an ongoing discussion to support installing from `.zip` natively with winget in [this issue](https://github.com/microsoft/winget-cli/issues/140), along with recent activity.

Alternatively:

- there was also some discussion in this [goreleaser issue](https://github.com/goreleaser/goreleaser/issues/1295), but that one does not seem very active.
- we could create a very simple installer using something like NSIS (but I am not sure how well that can integrate with our github actions workflows atm)

In general, I do like the idea of having `winget install ttn-lw-cli`, but it is currently not as straightforward as declaring a YAML manifest. My suggestion would be to wait until [Winget adds support for installing from .zip files](https://github.com/microsoft/winget-cli/issues/140).

If this takes a long time, and people from the community want this sooner, then we can prioritize this and see what the best way would be to create a simple installer for ttn-lw-cli.

**In short, this issue is currently blocked. If you are part of the community and would like `ttn-lw-cli` to get winget support sooner, please upvote this issue.**
